### PR TITLE
[JFR] Add compatibility for UnlockCommercialFeatures

### PR DESCRIPTION
**Pull request by D-D-H**
Summary: add compatibility for UnlockCommercialFeatures

Reviewed-by: yfxhust, luchsh

### CODE REVIEW

File: src/share/vm/jfr/dcmd/jfrDcmds.cpp [hunk 1]
```diff
@@ -626,6 +626,8 @@ bool register_jfr_dcmds() {
   DCmdFactory::register_DCmdFactory(new DCmdFactoryImpl<JfrStartFlightRecordingDCmd>(full_export, true, false));
   DCmdFactory::register_DCmdFactory(new DCmdFactoryImpl<JfrStopFlightRecordingDCmd>(full_export, true, false));
   DCmdFactory::register_DCmdFactory(new DCmdFactoryImpl<JfrConfigureFlightRecorderDCmd>(full_export, true, false));
+
```
Comment (yfxhust) [inline src/share/vm/jfr/dcmd/jfrDcmds.cpp]: An empty line.
Please remove it.

File: src/share/vm/jfr/dcmd/jfrDcmds.hpp [hunk 2]
```diff
@@ -166,6 +166,25 @@ class JfrConfigureFlightRecorderDCmd : public DCmdWithParser {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
+class JfrUnlockCommercialFeaturesDCmd : public DCmd {
+public:
+  JfrUnlockCommercialFeaturesDCmd(outputStream* output, bool heap) : DCmd(output, heap) { }
+  static const char* name() { return "VM.unlock_commercial_features"; }
+  static const char* description() {
+    return "Simulate commercial features unlocking for Zulu.";
```
Comment (yfxhust) [inline src/share/vm/jfr/dcmd/jfrDcmds.hpp]: Maybe we should refine it as "Simulate commercial features unlocking for Alibaba Dragonwell8."

**Comment by luchsh** [comment]
Please add a JTreg test to cover this change, same as AJDK, we should only merge code changes with testcases.

**Comment by luchsh** [comment]
Please follow the commit_message_template by adding a link to the Github issue this patch tries to fix

**Comment by D-D-H** [comment]
> Please add a JTreg test to cover this change, same as AJDK, we should only merge code changes with testcases.

done

> Please follow the commit_message_template by adding a link to the Github issue this patch tries to fix

done

thx

**Pull request merged**